frontend/src/metabase/containers/AddToDashSelectDashModal/AddToDashSelectDashModal.unit.spec.tsx

```diff
@@ -408,6 +407,95 @@ describe("AddToDashSelectDashModal", () => {
               }),
             ).toBeInTheDocument();
           });
+
+          describe('when "Create a new dashboard" option is clicked', () => {
+            beforeEach(async () => {
+              userEvent.click(
```

Comment:
AFAIK this is a rather uncommon usage of `beforeEach` in our codebase.

We could put the following code here as well:

```js
              // Create a new dashboard modal
              expect(
                screen.getByRole("heading", { name: "New dashboard" }),
              ).toBeInTheDocument();
              userEvent.click(screen.getByTestId("select-button"));
```

or, to make things consistent, we could remove this `beforeEach` and inline this code into respective tests.

This comment applies to the other `describe` as well.